src/ConnectionErrorModal.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import React from 'react';
+import { Button, Col, Modal, Row } from 'react-bootstrap';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import { ReactComponent as ErrorIcon } from './svg/Error.svg';
+import './ConnectionErrorModal.css';
+
+type ConnectionErrorModalProps = {
+    intl: any,
+    show: boolean,
+    onCancel: () => void,
+    onRetry: () => void
+};
+
+class ConnectionErrorModal extends React.Component<ConnectionErrorModalProps, {}> {
```

Comment:
The messaging on this component is Dash-specific, so let's rename the component to `DashConnectionErrorModal`.